community/blog/2023-01 2022, the year that Arcus put on grown-up pants/README.md

```diff
@@ -0,0 +1,45 @@
+# 2022, the year that Arcus put on grown-up pants
+Arcus has never grown in releases as much as in 2022. Come and see this post on the biggest changes on the Arcus front. It is time to put on grown-up pants.
+
+Arcus has grown on all fronts this year. Not only has it 22 releases, but it also received 17 issues from 10 external contributors and 11 pull requests from 6 external members. This is a very nice number and a great improvement from last year.
+
+![Arcus stats](media/arcus-stats.png)
+
+We tried this year to place Arcus more in the spotlight. Releasing more frequently is one of the ways we did that. In those 22 releases are some smaller ones but each has much added value and provides real production value for your project. A higher release rate also means that we can respond to certain problems more quickly. We are seeing Arcus being adapted and invested in, which results in more issues and pull requests. More and more, Arcus receives the recognition it deserves.
+
+## Service-to-service correlation
+Correlation was a hot topic this year. Service-to-service correlation was a feature that was striving for and that we not only implemented across all Arcus repositories but even have improved upon. The deprecated hierarchical correlation system was implemented in the first round but currently, we are already moving toward the new W3C correlation system. This new system allows us to interact more with Microsoft functionality like automatically tracking dependencies that are interacted with Microsoft technology (Azure SDK, HTTP, SQL). Having automatic dependency tracking is a big selling point for Arcus as it builds upon existing Microsoft functionality without the need for the consumer to move away from its features. In essence, that is what Arcus is all about: improving existing technology.
+
+🚩 [More info on Arcus HTTP correlation](https://webapi.arcus-azure.net/features/correlation/) 
+
+## Secret management
+The Arcus secret store is a great Arcus feature that lets you store secrets more safely in a dedicated store with a single interaction point instead of being scattered in the already populated application configuration. This highly-extensible feature is one of the oldest features in our repertoire and just got more amazing this year.
+
+Versioned and synchronous secret retrieval were added to the Arcus secret store this year. Secret versions allow you to store multiple versions of a single secret in your registered secret providers. Synchronous secret retrieval is an alternative way of secret retrieval instead of the built-in supported asynchronous way. Both are big changes internally but rather small outward changes. Both were grown from consumer interaction with the secret store, making this great feature all the more greater.
```

Comment:
```suggestion
Versioned and synchronous secret retrieval was added to the Arcus secret store this year. Secret versions allow you to store multiple versions of a single secret in your registered secret providers. Synchronous secret retrieval is an alternative way of secret retrieval instead of the built-in supported asynchronous way. Both are big changes internally but rather small outward changes. Both were grown from consumer interaction with the secret store, making this great feature all the more greater.
```